src/clients/node/src/benchmark.ts

```diff
@@ -4,19 +4,20 @@ import {
   createClient,
   Transfer,
   TransferFlags,
-  CreateTransfersError,
-  Operation
 } from '.'
 
 const MAX_TRANSFERS = 51200
 const MAX_REQUEST_BATCH_SIZE = 5120
 const IS_TWO_PHASE_TRANSFER = false
-const PREVIOUS_BENCHMARK = IS_TWO_PHASE_TRANSFER ? 150000 : 310000
-const TOLERANCE = 10 // percent that the benchmark is allowed to deviate from the previous benchmark
+
+const getEnvVar = (name: string): string => {
+  if (!process.env[name]) throw new Error(`environment variable ${name} required`)
+  return process.env[name] as string
+}
 
 const client = createClient({
   cluster_id: 0n,
-  replica_addresses: ['3001']
+  replica_addresses: [getEnvVar('TB_ADDRESS')]
```

Comment:
nit: can we default to 3000 if no environment variable was found?
The other clients do that.

Reply:
Was thinking about it, but presumed it could hamper debugging. If other clients already do so, it should be fine.